app/src/main/java/com/jnj/vaccinetracker/visitsoverview/screens/VisitsListFragment.kt

```diff
@@ -183,8 +188,36 @@ class VisitsListFragment(private val visitsKey: String) : BaseFragment(),
             workbook.write(outputStream)
             workbook.close()
         }
+
+        openExcelFile(file)
```

Comment:
opening a file is called in exportToFile method so it shouldn't be called once again here